Feature/kamil.modal buy

ssszafir opened the pull request.

DamianSikorski committed:
Update gallery.html

KitsuneAkvma committed:
Merge pull request #13 from KitsuneAkvma/feature/gallery

Feature/gallery

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
slider added

KitsuneAkvma committed:
UPDATE LOCAL HOST

DorotaDomanska committed:
Hero desktop

DorotaDomanska committed:
Hero desktop

KitsuneAkvma committed:
Merge pull request #61 from KitsuneAkvma/dev-merge

Main major merge

KitsuneAkvma committed:
Update package.json

ssszafir committed:
update

Na razie nie ma przycisku buy więc modal został podpięty pod przycisk product

ssszafir committed:
update

Na razie nie ma przycisku buy więc modal został podpięty pod przycisk product

ssszafir committed:
update

Na razie nie ma przycisku buy więc modal został podpięty pod przycisk product

ssszafir committed:
update

Na razie nie ma przycisku buy więc modal został podpięty pod przycisk product

ssszafir committed:
update button buy now

ssszafir committed:
update button buy now

ssszafir committed:
update button buy now

ssszafir committed:
update button buy now

ssszafir committed:
update

ssszafir committed:
update

ssszafir committed:
update

ssszafir committed:
update

kwiatkowsko reviewed (changes requested):
Omówione na czacie poprawki.

KitsuneAkvma committed:
Delete README.md

KitsuneAkvma committed:
reinstal gh-pages

KitsuneAkvma committed:
Update package.json

KitsuneAkvma committed:
scss rebuild

KitsuneAkvma committed:
przywrócenie backupu przed reinstallem

KitsuneAkvma committed:
Update package.json

KitsuneAkvma committed:
Update package.json

KitsuneAkvma committed:
Przywrócenie starej wersji

MalgorzataMarczynska committed:
poprawka buga ze zdjęciem krowy

ssszafir committed:
update

ssszafir committed:
update

ssszafir committed:
update

ssszafir committed:
update

KamilSzafranski commented:
> Omówione na czacie poprawki.

Poprawione

KitsuneAkvma committed:
reviews slide structure update

DorotaDomanska committed:
Drobne poprawki do hero desktop

DorotaDomanska committed:
Drobne poprawki do hero desktop

kwiatkowsko committed:
add hover and transition

kwiatkowsko committed:
add hover and transition

kwiatkowsko committed:
Update _setup.scss

kwiatkowsko committed:
Update _setup.scss

kwiatkowsko committed:
Merge pull request #65 from KitsuneAkvma/bugfix/cow-image

fix image-cow-desktop

KitsuneAkvma committed:
Okiełznano stukturę, style i  chowanie reszty elementów

kwiatkowsko committed:
Merge branch 'feature/DorotaDomanska/hero__desktop' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/DorotaDomanska/hero__desktop

kwiatkowsko committed:
Update hero.html

kwiatkowsko committed:
Merge pull request #60 from KitsuneAkvma/feature/DorotaDomanska/hero__desktop

Hero desktop

kwiatkowsko committed:
Update _contacts.scss

KitsuneAkvma reviewed (changes requested):
- Przydałby się jakiś kontener żeby to nie było tak blisko granicy

![obraz](https://user-images.githubusercontent.com/106407746/188745881-121f5e0c-57e1-4b18-a6f7-b1b12c15c8af.png)

- Chose na choose :) 
![obraz](https://user-images.githubusercontent.com/106407746/188746079-c9313194-4f2d-42b4-9a8f-f40da3f824ca.png)

- Usnunąć upper case - średnio pasuje do czcionki i fontu 
![obraz](https://user-images.githubusercontent.com/106407746/188747041-8ff718be-6d5a-4bed-b6ad-88f45488feb2.png)

- Zwiększyć gapy pomiędzy labelami i zwiększyć trochę kontrast pól bo się to troszkę zlewa 
![obraz](https://user-images.githubusercontent.com/106407746/188747196-caaddf6d-32b4-498b-ba1d-b8ec85264f65.png)

- Select effect też do poprawki bo się nie wyróżnia za bardzo ( może inner shadow)
![obraz](https://user-images.githubusercontent.com/106407746/188747271-90dbafa7-d338-4cae-9e52-53d8596fd2c4.png)

- Button powinien mieć 2rem radius
![obraz](https://user-images.githubusercontent.com/106407746/188747578-2dc20b01-2a1a-4d04-b84b-83c2ab02699b.png)

- Button w mobile-menu nie działa 
![obraz](https://user-images.githubusercontent.com/106407746/188747665-9bda7e17-51f4-4155-ab05-3f87bee2ad14.png)


Ogólnie to dobra robota Kamil. Fajnie to wygląda, podziwiam za chęci.🔥🔥🔥🔥

kwiatkowsko committed:
Merge branch 'feature/Agata/hover-and-shadow' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/Agata/hover-and-shadow

KitsuneAkvma committed:
Merge pull request #66 from KitsuneAkvma/feature/Agata/hover-and-shadow

Agata/Contacts/desktop + hover patch

kwiatkowsko committed:
Merge branch 'main' into dev-merge/Agata

KitsuneAkvma committed:
Merge pull request #67 from KitsuneAkvma/dev-merge/Agata

Dev merge/agata

KitsuneAkvma committed:
Działający slider

Kod jest toporny ale udało mi się to skleić. Niestety z moją mierną znajomością JS tyle udało mi się zrobić

KitsuneAkvma committed:
Merge pull request #68 from KitsuneAkvma/Reviews/slider

Reviews/slider added, functioning, no conflict

KitsuneAkvma committed:
Imiona i lokalizacje, aktualizują się  teraz

KitsuneAkvma committed:
Merge pull request #69 from KitsuneAkvma/Reviews/slider

Imiona i lokalizacje teraz się aktualizują

KitsuneAkvma committed:
Merge pull request #70 from KitsuneAkvma/dev-merge

Dev merge 3

KitsuneAkvma committed:
update komentarzy

lepiej wygląda :)

KitsuneAkvma committed:
Aktualizacja tytułu wg zadań technicznych

ssszafir committed:
Merge branch 'feature/kamil.modal-buy' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/kamil.modal-buy

kwiatkowsko committed:
add png light, customize >, fix margin-contacts

kwiatkowsko committed:
add png light, customize >, fix margin-contacts

ssszafir committed:
update

ssszafir committed:
update

ssszafir committed:
update

ssszafir committed:
update

kwiatkowsko committed:
Merge branch 'fix/Agata/margin-contacts-and-optima-png' of https://github.com/KitsuneAkvma/tankteam-project-1 into fix/Agata/margin-contacts-and-optima-png

kwiatkowsko committed:
fix icon-button

DorotaDomanska committed:
Merge pull request #71 from KitsuneAkvma/fix/Agata/margin-contacts-and-optima-png

Fix/agata/margin contacts and optima png

MalgorzataMarczynska committed:
dodanie favicon

DorotaDomanska committed:
animation

DorotaDomanska committed:
animation

kwiatkowsko committed:
Merge pull request #72 from KitsuneAkvma/feature/favicon

dodanie favicon

DorotaDomanska committed:
Merge branch 'feature/DorotaDomanska/animation-hero' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/DorotaDomanska/animation-hero

kwiatkowsko committed:
Merge pull request #73 from KitsuneAkvma/feature/DorotaDomanska/animation-hero

hero-circle-animation

kwiatkowsko commented:
Świetna robota!
Przepraszam, że dopiero teraz sprawdzam, ostatnia kosmetyka:
- country zgubiło literkę r - chyba, że chodziło Ci o wpisanie hrabstwa :p
-  &:focus-within {
      box-shadow: getColor(accent) 0px 2px 5px -1px, getColor(accent) 0px 1px 3px -1px;
    } albo zmień promień z 5px na 2px albo kolor na secondary-pink - ale to już kosmetyka
- możnaby odsunąć tego X troszeczkę, ale to też kosmetyka
- musisz mi zrobić rebase, bo zaszło za dużo zmian, a sama tego nie zrobię, bo nie wiem co w Twoim kodzie powinno zostać :p

Stopka świetna, dokładnie tak miała wyglądać :)

ssszafir committed:
Merge branch 'feature/kamil.modal-buy' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/kamil.modal-buy

ssszafir committed:
update

KamilSzafranski commented:
> Świetna robota! Przepraszam, że dopiero teraz sprawdzam, ostatnia kosmetyka:
> 
> * country zgubiło literkę r - chyba, że chodziło Ci o wpisanie hrabstwa :p
> * &:focus-within {
>   box-shadow: getColor(accent) 0px 2px 5px -1px, getColor(accent) 0px 1px 3px -1px;
>   } albo zmień promień z 5px na 2px albo kolor na secondary-pink - ale to już kosmetyka
> * możnaby odsunąć tego X troszeczkę, ale to też kosmetyka
> * musisz mi zrobić rebase, bo zaszło za dużo zmian, a sama tego nie zrobię, bo nie wiem co w Twoim kodzie powinno zostać :p
> 
> Stopka świetna, dokładnie tak miała wyglądać :)

zrobione

kwiatkowsko reviewed (approved):
cudownie, puszczam
